### ISSUE TITLE
[Feature Request] Access/Auto Hide second-sidebar while on Fullscreen

### ISSUE THREAD

**Issue by randomhero0** [aminought/firefox-second-sidebar#39]
I work mostly fullscreen and it would be great if I could access the second-sidebar by hovering the edge of the screen rather than hitting F11 each time. Is this possible? Either way, thank you for the hard work. The sidebar is truly amazing!

**Comment by VlaK0r** [comment]
Yes, it would be very cool if the panel had an auto-hide feature!
